internal/postprocessor/main.go

```diff
@@ -157,6 +162,8 @@ func (c *config) run(ctx context.Context) error {
 	if _, err := c.Manifest(generator.MicrogenGapicConfigs); err != nil {
 		return err
 	}
+	// TODO(codyoss): In the future we may want to make it possible to be able
+	// to run this locally with a user defined remote branch.
```

Comment:
Is this different than the branchOverride flag?

Reply:
Maybe I should have put more in the todo, but it does not work today because of the writting to the file thing. We would need to add logic to push the updates.

Reply:
Oh right I had forgotten about that. Okay looks good!